scripts/bootstrap-container

```diff
@@ -41,6 +41,13 @@ function run() {
 	esac
 }
 
+function k8s_build() {
```

Comment:
please apply [shfmt](https://github.com/mvdan/sh).

@szkl seems we need ci/check-lint from packer repo here as well...

Reply:
Noted.